src/main/java/racingcar/model/Cars.java

```diff
@@ -0,0 +1,56 @@
+package racingcar.model;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.NoSuchElementException;
+import java.util.Objects;
+import java.util.stream.Collectors;
+
+public final class Cars {
+    private final List<Car> cars;
+
+    public Cars(List<Car> cars) {
+        validate(cars);
+        this.cars = new ArrayList<>(cars);
+    }
+
+    private void validate(List<Car> cars) {
+        Objects.requireNonNull(cars, "전달된 리스트가 올바르지 않습니다 : Cars is null");
+
+        if (cars.isEmpty()) {
+            throw new IllegalArgumentException("전달된 리스트가 올바르지 않습니다 : Cars is empty");
+        }
+    }
+
+    public Cars play(MovingStrategy movingStrategy) {
+        List<Car> result = new ArrayList<>();
+        for (Car car : cars) {
+            result.add(car.move(movingStrategy));
+        }
+        return new Cars(result);
```

Comment:
다른 메서드들도 FP를 잘 활용해서 메서드 레벨로 Java8 Stream을 잘 활용해주셨는데요.
이 메서드도 한 번 시도해보시면 좋을 것 같습니다. 😄 
```suggestion
        return cars.stream()
                .map(car -> car.move(movingStrategy))
                .collect(Collectors.collectingAndThen(Collectors.toList(), Cars::new));
```

Reply:
우앗 이렇게 Stream으로 바로 생성할 수 있군요..!! 감사합니다 +_+